cudapoa/src/cudapoa_batch.cpp

```diff
@@ -91,10 +91,11 @@ CudapoaBatch::CudapoaBatch(int32_t max_sequences_per_poa,
                                   cuda_banded_alignment))
     , max_poas_(batch_block_->get_max_poas())
 {
+    scoped_device_switch dev(device_id_);
+
     bid_ = CudapoaBatch::batches++;
 
     // Set CUDA device
```

Comment:
this comment should be removed, too...

Reply:
Moved the comment to the scoped device switch call